rpm/podman.spec

```diff
@@ -238,12 +238,8 @@ LDFLAGS="-X %{ld_libpod}/define.buildInfo=${SOURCE_DATE_EPOCH:-$(date +%s)} \
          -X %{ld_libpod}/config._etcDir=%{_sysconfdir} \
          -X %{ld_project}/pkg/systemd/quadlet._binDir=%{_bindir}"
 
-%if %{defined copr_build}
-# ##GIT_COMMIT## is set by `.packit.sh` in Packit's Copr RPM build jobs.
-# Has no effect on Koji builds.
```

Comment:
This should not be removed, reader still need to know what fills that value in.

I don't like `##GIT_COMMIT##` part, it means we put a total invalid value in there if the replacement fails and we don't notice it right away. I think just setting `GIT_COMMIT=` (i.e. empty) is better and then just sed that part.

Reply:
Done. PTAL